.travis.yml

```diff
@@ -1,11 +1,7 @@
 language: ruby
 rvm:
-  # The version used at SLE12
-  - 2.1.2
 
   # Stable versions
-  - 2.1.10
-  - 2.2.6
```

Comment:
The changes on this file are unrelated to this PR. Moreover, why should we stop testing these versions ? (maybe we could upgrade the 2.2.6 to 2.2.7)